example/scalalib/testing/6-test-group-stealing/build.mill

```diff
@@ -0,0 +1,58 @@
+// This example shows how to use test grouping in combination with test stealing.
+
+//// SNIPPET:BUILD1
+package build
+import mill._, scalalib._
+
+object foo extends ScalaModule {
+  def scalaVersion = "2.13.8"
+  object test extends ScalaTests {
+    def ivyDeps = Seq(ivy"com.lihaoyi::utest:0.8.5")
+    def testFramework = "utest.runner.Framework"
+
+    def testForkGrouping = discoveredTestClasses().grouped(10).toSeq
+    def useTestStealingScheduler = true
+  }
+}
+
+/** See Also: foo/test/src/GroupX1.scala */
+/** See Also: foo/test/src/GroupX2.scala */
+/** See Also: foo/test/src/GroupX3.scala */
+/** See Also: foo/test/src/GroupX4.scala */
+/** See Also: foo/test/src/GroupX5.scala */
+/** See Also: foo/test/src/GroupX6.scala */
+/** See Also: foo/test/src/GroupX7.scala */
+/** See Also: foo/test/src/GroupX8.scala */
+/** See Also: foo/test/src/GroupX9.scala */
+/** See Also: foo/test/src/GroupX10.scala */
+/** See Also: foo/test/src/GroupY1.scala */
+/** See Also: foo/test/src/GroupY2.scala */
+/** See Also: foo/test/src/GroupY3.scala */
+/** See Also: foo/test/src/GroupY4.scala */
+/** See Also: foo/test/src/GroupY5.scala */
+/** See Also: foo/test/src/GroupY6.scala */
+/** See Also: foo/test/src/GroupY7.scala */
+/** See Also: foo/test/src/GroupY8.scala */
+/** See Also: foo/test/src/GroupY9.scala */
+/** See Also: foo/test/src/GroupY10.scala */
```

Comment:
`See Also` inlines the referenced file in the docs. I think we can just elide all of this; there's a `browse` link anyway anyone who wants can still get to the detailed sources